IdentityCore/src/auth_scheme/MSIDAuthenticationScheme.h

```diff
@@ -44,7 +44,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSDictionary *schemeParameters;
 @property (nonatomic, readonly) MSIDCredentialType credentialType;
 @property (nonatomic, readonly) NSString *tokenType;
-@property (nonatomic, readonly) MSIDAccessToken *accessToken;
+@property (nonatomic, readonly) MSIDAccessToken *blankAccessToken;
```

Comment:
nit : I think I like the previous naming better. Or maybe we cam come with something like "rawAccessToken"

Reply:
Addressed, I will just change back to accessToken.